sdk/resources/arm-resources/LICENSE.txt

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
@dw511214992 Why was this file deleted?

Reply:
It seem I missed a parameter when run autorest command. I will add it.